README.md

```diff
@@ -7,19 +7,22 @@ Yet another F-Droid client.
 
 ## Description
 
-Unofficial F-Droid client that resembles the classic F-Droid client.
+Unofficial F-Droid client resembling the classic F-Droid client.
 
-The client supports all basic F-Droid features, such as repository management and package installation. It doesn't
-support privileged extension, root installation, or local repositories.
+Jump over the lazy dog, manage repositories and install software quickly.
+No privileged extension, root installation, or sharing local repositories nearby yet.
 
-### Features
+## Features
 
-* Classic F-Droid style, without cards or inappropriate animations
-* Fast repositories synchronization under good network conditions
-* Built upon standard Android components and a minimal set of dependencies
-* Made with attention to details, both in program logic and design
+* Classic F-Droid style.
+* No cards or inappropriate animations.
+* All apps shown in "Available".
+* Fast repository syncing.
```

Comment:
I though it's actually important to notice about "network conditions". F-Droid servers like to hang sometimes, so everything is slow as hell, and it's not application-dependent.

Reply:
Then throw an error that suggests the culprit is F-Droid upstream.
If the user wants to decide whether to use Foxy Droid, things like this are just superfluous.
F-Droid itself doesn't mention it, and the user isn't getting away from it between the two.
Nobody is going to hold it against Foxy Droid that this was not mentioned. It is a fatalist sentence that does not belong in the prominent features.

Reply:
Ok.